Simplenote/SharedStorageMigrator.swift

```diff
@@ -0,0 +1,154 @@
+import Foundation
+import CoreData
+
+class SharedStorageMigrator {
+
+    static let documentsURL: URL? = {
+        FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
```

Comment:
```suggestion
        FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!
```

Reply:
This is a... freedom we usually take. There will always be a Document Directory, if not, we've got a much bgger problem. (Plus it solves all of the `?` all over)